app.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "meanjs",
+  "description": "Full-Stack JavaScript with MongoDB, Express, AngularJS, and Node.js",
+  "repository": "https://github.com/Ghalleb/mean.git",
+  "logo": "http://meanjs.org/img/logo-small.png",
+  "keywords": [
+    "mongodb",
+    "express",
+    "angularjs",
+    "node.js",
+    "mongoose",
+    "passport"],
```

Comment:
I took the ones frome here:
https://github.com/meanjs/mean/blob/master/config/env/default.js